pydra/engine/helpers_file.py

```diff
@@ -0,0 +1,479 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+
+""" functions copied from nipype, removed parts that were related to py2"""
+
+import subprocess as sp
+import hashlib
+from hashlib import md5
+import os
+import os.path as op
+import re
+import shutil
+import posixpath
+from builtins import str, bytes, open
+
+
+related_filetype_sets = [(".hdr", ".img", ".mat"), (".nii", ".mat"), (".BRIK", ".HEAD")]
```

Comment:
perhaps add a note here that these are imaging specific, but may need to be generalized for hash computation and for copyfile.

Reply:
ok